pybrake/test_queries.py

```diff
@@ -0,0 +1,75 @@
+import time
+
+from .queries import QueryStat, QueryStats
+
+
+def test_query_metric_start_end():
+    stats = QueryStat(time=1648551580.0367732)
+    assert stats.__dict__ == {
+        'count': 0,
+        'sum': 0,
+        'sumsq': 0,
+        'tdigest': 'AAAAAkAkAAAAAAAAAAAAAA==',
+        'query': '',
+        'method': '',
+        'route': '',
+        'time': '2022-03-29T10:59:00Z'
+    }
+
+
+def test_query_stats_performance_stats():
+    stats = QueryStats(**{"config": {
+        "performance_stats": False,
+        "error_host": "https://api.airbrake.io",
+        "apm_host": "https://api.airbrake.io",
+    }})
+    assert stats.notify(
```

Comment:
What does this actually test? That `notify` returns `None`? How does this differ from the test on line 35 (`test_query_stats_query_stats`)? I can see exactly the same assertion is made.

Reply:
Understood, thanks. My suggestion would be making the test name reflect this scenario.

Reply:
It is already the case. But I'm still going to add the docstring.